readme.md

```diff
@@ -473,6 +473,7 @@
 - [Alfred Workflows](https://github.com/alfred-workflows/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
 - [GitHub Actions](https://github.com/sdras/awesome-actions#readme) - Create tasks to automate your workflow and share them with others on GitHub.
+- [Version Managers](https://github.com/bernardoduarte/awesome-version-managers) - Version managers for usage of multiple versions of the same programming language.
```

Comment:
The description needs to be better written. It also doesn't need to repeat "version managers".

You also forgot `#readme`.

Reply:
@sindresorhus Got it! I've updated the description and added the #readme to the URL.

Also, I have to point out that it is not stated on the guidelines that the entry should have a #readme on it, I think it would be good to add to it.